### PR TITLE
EDU-765 Display comments on doc page

### PR DESCRIPTION
**Pull request by adrianaluchian**
Closes https://educandu.atlassian.net/browse/EDU-765

**Desktop:**

Screen Shot 2022-08-31 at 17 08 17

**Mobile:**

Screen Shot 2022-08-31 at 17 14 02

### CODE REVIEW

File: src/components/pages/doc.yml [hunk 1]
```diff
@@ -10,3 +10,6 @@ documentMetadataUpdated:
 commentsPanelTitle:
   en: You are viewing the document page in "comments" mode
   de: Du befindest dich im Kommentar-Modus
+commentsHeader:
+  en: Comment topics
+  de: Kommentare-Themen
```
Comment (ahelmberger) [inline src/components/pages/doc.yml:15]: No double-plural here, just "Kommentar-Themen"

**Comment by ahelmberger** [comment]
> Closes https://educandu.atlassian.net/browse/EDU-765
> 
> **Desktop:**
> 
> Screen Shot 2022-08-31 at 17 08 17
> 
> **Mobile:**
> 
> Screen Shot 2022-08-31 at 17 14 02

Nice layout :heart:

File: src/services/client-data-mapping-service.js [hunk 2]
```diff
@@ -190,18 +190,22 @@ class ClientDataMappingService {
     return mappedUserFavorites;
   }
 
-  mapComment(comment) {
+  async mapComment(comment) {
     const mappedComment = cloneDeep(comment);
+    const rawUser = await this.userStore.getUserById(comment.createdBy);
```
[the remark this answers is not in the document]
Comment (adrianaluchian) [reply src/services/client-data-mapping-service.js:195]: Of course, a very valid point, I will change this.

**Comment by adrianaluchian** [comment]
> > [...]
> 
> Nice layout ❤️

Thanks 🙇 The accordion part I'm not really happy with, from UI perspective, but we can keep this until a styling idea comes along.

File: src/components/pages/doc.yml (hunk 1 above)
Comment (adrianaluchian) [reply src/components/pages/doc.yml]: Corrected, thank you

File: src/services/client-data-mapping-service.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (adrianaluchian) [reply src/services/client-data-mapping-service.js]: Fixed, thanks!

**Review by ahelmberger — APPROVED** [review]

**Pull request merged**